offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/AppPresenter.groovy

```diff
@@ -26,4 +27,9 @@ class AppPresenter implements UpdateCustomerOutput {
     void failNotification(String notification) {
         viewModel.failureNotifications.add(notification)
     }
+    
+    @Override
```

Comment:
I was not sure why the AppPresenter implements this interface, so I just fixed it like this.

Reply:
I think this is not what we want to have. The UpdateCustomerPresenter/CreateCustomerPresenter should do that. I think you should just remove the implementing interface here.

Reply:
Thanks, done